scripts/ensure_ollama_resources.py

```
@@ -0,0 +1,127 @@
+    payload = {
+        "model": model,
+        "prompt": 'Return JSON: {"ok": true}',
+        "stream": False,
+        "format": "json",
+        "options": {"temperature": 0.0, "num_predict": 64},
+    }
+    try:
+        resp = httpx.post("http://localhost:11434/api/generate", json=payload, timeout=60)
+        if resp.status_code != 200:
+            return False, f"status={resp.status_code}, body={resp.text[:200]}"
+        return True, "ok"
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)